x-pack/plugins/security_solution/server/lib/detection_engine/signals/executors/threshold.ts

```diff
@@ -83,7 +77,7 @@ export const thresholdExecutor = async ({
       services,
       logger,
       ruleId: ruleParams.ruleId,
-      bucketByFields: normalizeThresholdField(ruleParams.threshold.field),
```

Comment:
Sanity check: this is no longer necessary due to the migration, correct?

Reply:
Right, the migration will convert all `threshold.field` values to `string[]`. By this point in the executor we've runtime-validated with `thresholdSpecificRuleParams` so the compiler knows `ruleParams.threshold` must be of type `ThresholdNormalized`